middleware/authorize.js

```diff
@@ -0,0 +1,34 @@
+const jwt = require("jsonwebtoken");
+
+const User = require("../models/user");
+
+const { SECRET_KEY } = process.env;
+
+const authorize = async (req, res, next) => {
```

Comment:
Hmm, ogólnie to działa. Ale chiałbym abyś zrobił to za pomocą strategii. Na lms masz w materiałach rozwiązane to za pomocą biblioteki passport, tam praktycznie jest podane rozwiązanie tylko musisz dorzucić tam sprawdzanie czy token z req jest taki sam jaki zwraca passport (czyli to co masz tutaj z tym "Bearer")

Reply:
wprowadziłem zmiany